packages/react/src/components/Dropdown/DropdownItem.tsx

```diff
@@ -102,6 +115,17 @@ class DropdownItem extends UIComponent<WithAsProp<DropdownItemProps>> {
             styles: styles.content,
           },
         })}
+        endMedia={
```

Comment:
This is different then how the rest of the elements are created and where the styles are applied, but there are issues with the height if we don't provide these styles.. I am open for other ideas...

Reply:
Ping @layershifter @miroslavstastny